examples/remote/server.rs

```diff
@@ -0,0 +1,52 @@
+//! A Bevy app that you can connect to with the BRP and edit.
```

Comment:
Since all that's required for an example to be compatible with the BRP is `.add_plugins(RemotePlugin::default())`, maybe this could be added to all the examples in `games` (feature gated perhaps?). Because the games are interactive, the querying via BRP will be more interesting IMO.

Reply:
I was actually thinking that maybe *all* examples should have it. But probably best to do that as a followup to keep the commits tidy.

Reply:
Alternatively, you could add `RemotePlugin` to the default plugins when in debug mode? That would add it to the examples and avoid accidentally leaking the ultimate cheat engine in production games.

Reply:
I think that should be a wider discussion after this PR lands, because I can imagine that this could be surprising for many. Opening ports without explicit developer consent makes me nervous.

Reply:
Can be in `DefaultPlugins` under a separate feature from just `bevy_remote`, then users can specify an additional feature to host a server

Reply:
Yeah, I think adding by default, but behind a feature flag is the solution, but like pcwalton said, this should be done at a later time in a follow up PR